lib/capistrano/scm/tasks/git.rake

```diff
@@ -70,4 +70,15 @@ namespace :git do
       end
     end
   end
+
+  desc "Determine the unix timestamp that the revision that will be deployed was created"
+  task :set_current_revision_time do
+    on release_roles(:all), in: :groups, limit: fetch(:git_max_concurrent_connections), wait: fetch(:git_wait_interval) do
+      within repo_path do
+        with fetch(:git_environmental_variables) do
+          set :current_revision, git_plugin.fetch_revision
```

Comment:
❓ I think there might be a typo here. Shouldn't it be:

```suggestion
          set :current_revision, git_plugin.fetch_revision_time
```

Reply:
whoops yeah good catch